DormService/Services/Payment/Payment.Common/Repository/DebtsRepository.cs

```diff
@@ -72,5 +75,19 @@ public async Task<bool> DeleteStudent(string studentID)
 
             return deleteResult.IsAcknowledged && deleteResult.DeletedCount > 0;
         }
+
+        public async Task<ReduceCreditDTO> GetStudentCredit(string studentID)
```

Comment:
I would suggest to return `StudentDebts` here and to map it into `ReduceCreditDTO` in the function below.